be/test/service/lake_service_test.cpp

```diff
@@ -396,6 +396,119 @@ TEST_F(LakeServiceTest, test_publish_version_for_write) {
     }
 }
 
+// NOLINTNEXTLINE
+TEST_F(LakeServiceTest, test_publish_version_for_write_batch) {
```

Comment:
Add some test cases to cover the switch between batch publish and single publish, with some txn logs been deleted.